keyboards/s7_elephant/keymaps/default/keymap.c

```diff
@@ -0,0 +1,95 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+enum custom_layers {
+    _QWERTY,
+    _FN
+};
+
+enum custom_keycodes {
+    EPRM = SAFE_RANGE,
+};
+
+#define _______ KC_TRNS
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31

```suggestion
```

Reply:
Fixed.